arduino/OpenMRN.h

```diff
@@ -465,25 +476,36 @@ class OpenMRN : private Executable
     }
 
 #if defined(ESP32)
+
+    /// Creates a FreeRTOS task on the ESP32 to automatically call the @ref loop
+    /// method. If the ESP32 hardware CAN device is in use the background task
+    /// is strongly suggested to be used. @ref start_background_task.
```

Comment:
The implementation of this is still broken (uses a fixed delay without taking into account whether there is work to be done). I would not recommend to use this now.

Reply:
Is there a way to detect if work is to be done? Or do you think the loop method can always be called?

Reply:
This won't be necessary as the BG task is going away. I'll have an update shortly with its removal.

Reply:
task has been removed.

Reply:
no longer relevant as the bg task has been removed.